reference/docs-conceptual/components/ise/object-model/The-ISEFileCollection-Object.md

```diff
@@ -1,22 +1,26 @@
 ---
-ms.date:  06/05/2017
+ms.date:  12/31/2019
 keywords:  powershell,cmdlet
 title:  The ISEFileCollection Object
 ---
+
 # The ISEFileCollection Object
 
-The **ISEFileCollection** object is a collection of **ISEFile** objects. An example is the $psISE.CurrentPowerShellTab.Files collection.
+The **ISEFileCollection** object is a collection of **ISEFile** objects. An example is the
+`$psISE.CurrentPowerShellTab.Files` collection.
 
 ## Methods
 
 ### Add\( \[fullPath\] \)
 
 Supported in Windows PowerShell ISE 2.0 and later.
 
-Creates and returns a new untitled file and adds it to the collection. The **IsUntitled** property of the newly created file is **$true**.
+Creates and returns a new untitled file and adds it to the collection. The **IsUntitled** property
+of the newly created file is `$true`.
 
-**\[fullPath\]** - Optional string
-The fully specified path of the file. An exception is generated if you include the **fullPath** parameter and a relative path, or if you use a file name instead of the full path.
+**\[fullPath\]** - Optional string The fully specified path of the file. An exception is generated
```

Comment:
Just to check, is the casing on that property correct?

Reply:
No it's not. Thank you and fixed.